Sprint-1/explore/chrome.md

```diff
@@ -12,7 +12,11 @@ invoke the function `alert` with an input string of `"Hello world!"`;
 
 What effect does calling the `alert` function have?
 
+<!-- a pop up alert with will show up on chrome displaying Hello world! -->
+
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
 What is the return value of `prompt`?
+
+<!-- prompt will ask for a prompt from the user by asking 'What is your name?'. The input is then added to the variable myName. The cancel button cancels the prompt. -->
```

Comment:
You can provide different messages for "OK" or "Cancel". e.g. "You canceled the prompt. No color provided." or "Your favorite color is ${userInput}."

Reply:
How can you tell if the user clicked the "Cancel" button or the "OK" button?

Reply:
You can use an if statement to determine if the user clicked "OK" or "Cancel".

Here is an example of how it will work.

let myName = prompt("What is your name?");
if (myName === null) {                                                                            // The user clicks cancel
    console.log("The user clicked Cancel.");
} else if (myName === "") {
    console.log("The user clicked OK but entered nothing.");             // The user clicks OK but doesn't enter anything
} else {
    console.log(`The user clicked OK and entered: ${myName}`);    // The user clicks OK
}